QUARKUS-5613.md

```diff
@@ -0,0 +1,85 @@
+## Links to the resources
+- JIRA issue: https://issues.redhat.com/browse/QUARKUS-5613
+- GH PRs related to the Jira issue: https://github.com/quarkusio/quarkus/pull/43283 https://github.com/quarkusio/quarkus/issues/42117 
+- Guides modified and related:
+  https://quarkus.io/guides/security-openid-connect-client-reference
+  https://quarkus.io/guides/security-oidc-code-flow-authentication
+  https://quarkus.io/guides/security-openid-connect-client-registration
+- TP/TD PRs:
+
+## Scope of the testing
+
+Our main testing scope is to ensure that the new response filter mechanism works as intended for various OIDC endpoints, ensuring that is possible intercept and handle responses
+from various OIDC endpoints.
+
+This complements the already available OidcRequestFilter functionality.
+
+Integration tests will be on the next module in our Test suite:    
+ security/keycloak-oidc-client-reactive-extended (https://github.com/quarkus-qe/quarkus-test-suite/tree/main/security/keycloak-oidc-client-reactive-extended).
+And the scope will be covered on areas related to:
+   - Security
+   - Logging /Tracing - Enabling TRACE level logs in properties will capture relevant info for our tests
+
+The test coverage will be verified for JVM, native mode, and Openshift.
+
+The testing include both OIDC Client (quarkus-oidc) and OIDC Client (quarkus-oidc-client) extensions.
+
+### What to Test - Scenarios
```

Comment:
In other words, you will test exactly what you see already tested in Quarkus test coverage, but there are other scenarios where response filter is applied like `TOKEN_REVOCATION`, `JWKS`, `INTROSPECTION`, ...

I can respect your choice to test exactly these 3, but I think it can't be just stated, it must be reasoned WHY you chose these and I don't see it in the TP. Please kindly say in the TP something like _I chose these 3 because..._